### ISSUE TITLE
tabView error

### ISSUE THREAD

**Issue by t3377806** [enisn/UraniumUI#250]
Hello,

I am writing to report an error that I encountered while trying to use the demo test application in the "test" folder. Specifically, I downloaded the application and I am using it in a Windows environment. Every time I click on a tab to switch pages, I receive four error messages that read as follows:

"Binding: 'IsSelected' property not found on 'UraniumUI.Material.Controls.TabItem', target property: 'Microsoft.Maui.Controls.Grid.Bound'"

I am not sure how to resolve this issue, and I would greatly appreciate any guidance or assistance that you can provide. Thank you for your help.

**Comment by enisn** [comment]
Possibly the same error with https://github.com/enisn/UraniumUI/issues/302#issuecomment-1537167082

There should be x:DataType when Compiled Bindings feature is used.

**Comment by garyvlc** [comment]
Hi @t3377806 , may i know whether you have resolved the issue? I am hitting the same error. 

@enisn Sorry, i am not really understand

**Comment by daredloco** [comment]
> Possibly the same error with [#302 (comment)](https://github.com/enisn/UraniumUI/issues/302#issuecomment-1537167082)
> 
> There should be x:DataType when Compiled Bindings feature is used.

The solution showed in the other post does not solve the problem and it's also a different error, as this happens without adding any custom binding or such.
The error happens, even if you use the first example in "Usage" here: https://enisn-projects.io/docs/en/uranium/latest/themes/material/components/TabView

```cs
<material:TabView>
    <material:TabItem Title="Tab One">
        <material:TabItem.ContentTemplate>
            <DataTemplate>
                    <Label Text="First Tab" />
            </DataTemplate>
        </material:TabItem.ContentTemplate>
    </material:TabItem>


    <material:TabItem Title="Tab Two">
        <material:TabItem.ContentTemplate>
            <DataTemplate>
                    <Label Text="Second Tab" />
            </DataTemplate>
        </material:TabItem.ContentTemplate>
    </material:TabItem>

    <material:TabItem Title="Tab Three">
        <material:TabItem.ContentTemplate>
            <DataTemplate>
                    <Label Text="Third Tab" />
            </DataTemplate>
        </material:TabItem.ContentTemplate>
    </material:TabItem>
</material:TabView>
```

**Comment by babenkovitaliy** [comment]
I am currently having the same issue. I am also seeing a performance drop when trying to click on each individual tab. The small catch with my setup is that I am using MacCatalyst and .NET 8.0, which is still in the preview stage (I decided to use the .NET 8.0 instead of the finalized .NET 7.0, because all of the .NET 7.0 bug fixes were being pushed to .NET 8.0 ONLY).

**Comment by enisn** [comment]
It seeems similar problem with https://github.com/enisn/UraniumUI/issues/141#issuecomment-1401547853

You can try using `xmlns:material="clr-namespace:UraniumUI.Material.Resources;assembly=UraniumUI.Material"` namespace instead

**Comment by Kumima** [comment]
Shouldn't the IsSelected property be bound in OneWay mode?

**Comment by babenkovitaliy** [comment]
Nope. Doesn't work. I tried using the suggestion above and then I tried to use the built-in autocomplete feature to populate the xmlns declaration and both returned the error below.

![image](https://github.com/enisn/UraniumUI/assets/69189298/df6e4aef-fd32-4fcd-8734-e32f7cde61ce)

**Comment by babenkovitaliy** [comment]
I think I found the source of the error. At the time of this writing, on [line number 492](https://github.com/enisn/UraniumUI/blob/develop/src/UraniumUI.Material/Controls/TabView.cs#L492), 'IsSelected' is not a property - it is a field/variable.

You'd need to change it to something like this (written manually, not tested):
`public bool IsSelected { get => TabView.SelectedTab == this || (TabView.CurrentItem != null && TabView.CurrentItem == Data); }`

A few notes about this suggestion:

- It does not have a setter because when looking through the code, I have not seen a single instance where 'IsSelected' has been set aside from the initial declaration.
- Converting it to a property should _hopefully_ solve the issue, but may potentially have a small side effect. The demo XAML pages that use this property may potentially throw an error if it attempts to set the property via binding. To solve this issue, you can change the binding to `{Binding Path=IsSelected, Mode=OneWay}`.
